docs/_user_guide/storage.md

```diff
@@ -1,276 +1,180 @@
 
-{% include warning.html content="Before transferring data to any platform, make sure the data is licensed for your use and that the storage platform meets the security requirements for that data." %}
+!!! Important
+    Before transferring data to any platform, make sure the data is licensed for your use and that the storage platform meets the [security requirements](/_policies/security/){:target="_blank"} for that data. 
 
 ## Preferred Platforms:
-- **ZFS:** The GSB's 612 TB high-performance storage solution. Currently, all Yen user and project directories reside on ZFS. Snapshots are taken of the data and are easily recoverable. ZFS is the ideal solution if you are performing your analysis or computational work on the Yens.
-- **Google Drive:** Available to all users at Stanford. [Approved](https://uit.stanford.edu/service/gsuite/drive) for low, medium and high risk data. Limited to 400,000 files and throttled to 750 GB upload per day. Ideal for audio, video, PDFs, Images, and flat files. Great for sharing with external collaborators. Also appropriate for [archiving research data](../faqs/rcloneGoogleDrive.html)
- - **Oak:** Similar to ZFS, "[Oak](https://uit.stanford.edu/service/oak-storage) is a High-Performance Computing (HPC) storage system available to research groups and projects at Stanford for research data." Oak is not snapshotted; deleted data is lost. It costs about $45 per 10 TB. If you expect to use [Sherlock](/training/14_sherlock_hpc.html), you will need to use Oak.
- - **Redivis:** [Redivis](https://redivis.com/) allows users to deploy datasets in a web-based environment (GCP backend) and provides a powerful query GUI for users who don't have a strong background in SQL. 
+- **ZFS:** The GSB's 1 PB high-performance storage solution. Currently, all Yen users and project directories reside on ZFS. Snapshots are taken of the data and are easily recoverable. ZFS is the ideal solution if you are performing your analysis or computational work on the Yens.
+- **Google Drive:** Available to all users at Stanford. [Google Drive](https://uit.stanford.edu/service/gsuite/drive){:target="_blank"} is approved for low, medium and high risk data. It supports up to 400,000 files and has a daily upload limit of 750 GB, making it ideal for storing audio, video, PDFs, images, and flat files. Google Drive is great for sharing with external collaborators and is also suitable for [archiving research data](/_user_guide/best_practices_archive/){:target="_blank"}
+ - **Oak:** Similar to ZFS, [Oak](https://uit.stanford.edu/service/oak-storage){:target="_blank"} is a High-Performance Computing (HPC) storage system available to research groups and projects at Stanford for research data. Oak does not support snapshots, so deleted data cannot be recovered. The cost is approximately $45 per 10 TB. If you expect to use [Sherlock](/_policies/sherlock/){:target="_blank"}, you will need to use Oak.
```

Comment:
Please clarify the $45 per 10 TB. Is that a one time cost or monthly cost?

Should we mention that Oak will be integrated to the Yens soon?

Reply:
I checked and it is a monthly cost. I added it to the oak section.

Reply:
I also added that Oak can be mounted on the Yens by request using an NFS gateway.